src/app/(pages)/codeLibrary/codeLibrary.module.scss

```diff
@@ -168,6 +168,16 @@
   }
 }
 
+.manageCodesLink {
+  font-weight: 700;
+  text-decoration: none;
+
+  &:active,
+  &:visited {
+    color: #0076d6;
```

Comment:
could we swap this hex code out with the relevant variable (or equivalent) from the new design system? alternatively in the JSX you can just use a `<Button unstyled />` setup